presto-product-tests/bin/run_on_docker.sh

```diff
@@ -114,42 +114,46 @@ function terminate() {
   exit 130
 }
 
+function getAvailableEnvironments() {
+AVAILABLE_ENVIRONMENTS=`ls $ENVIRONMENT_LOCATION -I files`
```

Comment:
you don't need to use the temporary variable. Just call the `ls $ENVIRONMENT_LOCATION -I files` within that function, and use it as $(getAvailableEnvironments).